UtiliKit.podspec

```diff
@@ -19,10 +19,10 @@ s.homepage          = 'https://github.com/BottleRocketStudios/iOS-UtiliKit'
 s.license           = { :type => 'Apache 2.0', :file => 'LICENSE' }
 s.author            = { 'Bottle Rocket Studios' => 'wilson.turner@bottlerocketstudios.com' }
 s.source            = { :git => 'https://github.com/bottlerocketstudios/iOS-UtiliKit.git', :tag => s.version.to_s }
-
+s.source_files      = 'Sources/UtiliKit/**/*'
+s.ios.deployment_target = '10.0'
```

Comment:
Is there a technical reason for updating the deployment target or do you just feel it's time to do so?

Reply:
In this case yes, `UIViewImplicitlyAnimating` 
https://developer.apple.com/documentation/uikit/uiviewimplicitlyanimating

Reply:
Sounds good, let's just make sure to call it out in the `CHANGELOG`.

Reply:
Yup, will also look to release as 2.0 as it's a major change.